src/components/forms/NetworkDevicesForm/InheritedNetworkRow.tsx

```diff
@@ -0,0 +1,146 @@
+}: Props): MainTableRow => {
+  const readOnly = (formik.values as EditInstanceFormValues).readOnly;
+  const overrideDevice = formik.values.devices.find(
+    (t) => t.name === device.key,
+  ) as LxdNicDevice | LxdNoneDevice;
+
+  const overrideNetwork = managedNetworks.find(
+    (t) =>
+      t.name ===
+      (formik.values.devices.find((t) => t.name === device.key) as LxdNicDevice)
+        ?.network,
+  );
+
+  const getProfileName = (device: InheritedNetwork) => {
+    if (device.source.endsWith(" profile")) {
+      return device.source.substring(0, device.source.indexOf(" profile"));
+    }
+
+    return device.source;
+  };
+
+  return getConfigurationRowBase({
+    configuration: (
+      <>
+        <b>{device.key}</b>
+      </>
+    ),
+    inherited: (
+      <div>
+        <div
+          className={classNames("p-text--small", "u-text--muted", {
+            "u-text--line-through": isOverridden,
+          })}
+        >
+          From profile{" "}
+          <ResourceLink
+            type="profile"
+            value={getProfileName(device)}
+            to={`/ui/project/${encodeURIComponent(project)}/profile/${encodeURIComponent(getProfileName(device))}`}
+            className={classNames({
+              "u-text--line-through": isOverridden,
+            })}
+          />
+        </div>
+        <div
+          className={classNames({
+            "u-text--muted": isOverridden,
+            "u-text--line-through": isOverridden,
+          })}
+        >
+          Network
+        </div>
+        <ResourceLink
+          type="network"
+          value={device.network?.network || ""}
+          to={`/ui/project/${encodeURIComponent(project ?? "")}/network/${encodeURIComponent(device.network?.network || "")}`}
+          className={classNames({
+            "u-text--line-through": isOverridden,
+          })}
+        />
+        <ReadOnlyAclsList
+          project={project}
+          network={managedNetworks.find(
+            (t) => t.name === device.network?.network,
+          )}
+          device={device.network}
+        />
+      </div>
+    ),
+    override: isOverridden ? (
+      <NetworkDevice
+        readOnly={readOnly}
+        formik={formik}
+        project={project}
+        focusNetwork={focusNetwork}
+        removeNetwork={removeNetwork}
+        device={overrideDevice}
+        network={overrideNetwork}
+        isOverridden
+        isInheritedRow
+      />
+    ) : (
+      <div className="network-device">
+        <div />
+        <NetworkDeviceActionButtons
+          readOnly={readOnly}
+          formik={formik}
+          onClickEdit={() => {
+            ensureEditMode(formik);
+            addDevice({
+              formik,
+              deviceName: device.key,
+              deviceNetworkName: managedNetworks[0]?.name ?? "",
+            });
+
+            focusNetwork(
+              formik?.values.devices.findIndex((t) => t.name === device.key),
+            );
```

Comment:
Similar to the other comment, I think this should focus the network selector, currently does not work as I'd expect.